modules/ROOT/partials/configuration/convert_fonts_to_spans.adoc

```diff
@@ -1,20 +0,0 @@
-[[convert_fonts_to_spans]]
```

Comment:
This wasn't deleted in the end (hence the strikethrough) and was delayed until 7.0 or later, so this shouldn't have been removed.